doc/source/autoscaling.rst

```diff
@@ -62,7 +67,7 @@ The setup commands you use should ideally be *idempotent*, that is, can be run m
 Syncing git branches
 --------------------
 
-A common use case is syncing a particular local git branch to all workers of the cluster. There is a nice way to do this as follows:
+A common use case is syncing a particular local git branch to all workers of the cluster. However, if you just put a `git checkout <branch>` in the setup commands, the autoscaler won't know when to rerun the command to pull in updates. There is a nice workaround for this by including the git SHA in the input (the hash of the file will change if the branch is updated):
```

Comment:
should maybe say something about head needing to be pushed on that branch

Reply:
Done